### ISSUE TITLE
Document's "content_uri" requirement not clear

### ISSUE THREAD

**Issue by gsora** [commercionetwork/commercionetwork#65]
## Summary of Bug

Documentation says that [`content_uri` can be omitted](https://docs.commercio.network/x/docs/tx/send-document.html#fields-requirements) if not required.

This is a snipped from the `Document` type declaration:

```go
type Document struct {
        // various fields...
	ContentURI     string                  `json:"content_uri"`     // Optional
        // ...other fields
}
```

The `Validate()` function in` document.go` says the opposite:

```go
if len(strings.TrimSpace(doc.ContentURI)) == 0 {
        return sdk.ErrUnknownRequest("Document content URI can't be empty")
}
```

## Version

`54014fb776482be13611bdf25eee969d88ce960b` on `version/1.3.2`.

## Steps to Reproduce

1. validate a `Document` with an empty `ContentURI`.

### Expected result

Validation continues and no `ContentURI`-related error gets returned.

### Actual result

Validation stops with the following error: `Document content URI can't be empty`.

**Comment by RiccardoM** [comment]
We should remove the check inside the `ValidateBasic` as the field is indeed optional

**Comment by RiccardoM** [comment]
Closes with #66
